### PR TITLE
TODO(sjmiles) cleanup

### PR DESCRIPTION
**Pull request by sjmiles**
Plan one more PR to attach Issue tracking to remaining TODOs.

### CODE REVIEW

File: src/tests/particles/multi-slot-test.ts [hunk 1]
```diff
@@ -1,92 +0,0 @@
-/**
- * @license
```
Comment (mmandlis) [inline src/tests/particles/multi-slot-test.ts]: should also delete the corresponding multi-slot-test.manifest file.

Comment (sjmiles) [reply src/tests/particles/multi-slot-test.ts]: Thank you, and thank you very much for the reviews in general.

**Comment by sjmiles** [comment]
Closing/reopening to trigger copybara tooling.

**Comment by sjmiles** [comment]
Can't get copybara to work, going to recreate PR.

**Pull request closed without merging**